docs-src/references/dynamic-configuration.md

```diff
@@ -0,0 +1,173 @@
+
+| Dynamic configuration key              | Type | Description                                                                                                                                                                                                                      | Default value |
+| -------------------------------------- | ---- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| Frontend                               |      |                                                                                                                                                                                                                                  |               |
+| `frontend.rps`                         | Int  | Rate limit per second. This value applies to each Frontend service host.                                                                                                                                                         | 2400          |
+| `frontend.namespaceRPS`                | Int  | Rate limit per second applied at each Namespace level.                                                                                                                                                                           | 2400          |
+| `frontend.globalNamespaceRPS`          | Int  | Namespace rate limit per second applied globally on the entire cluster. The limit is evenly distributed among available Frontend service instances. If this is set, it overrides the per-instance limit (frontend.namespaceRPS). | 0             |
+| `internal-frontend.globalNamespaceRPS` | Int  | Namespace rate limit per second across all internal-frontends.                                                                                                                                                                   | 0             |
+| History                                |      |                                                                                                                                                                                                                                  |               |
+| `history.rps`                          | Int  | Request rate per second for each history host.                                                                                                                                                                                   | 3000          |
+| `history.archiveRequestRPS`            | Int  | Rate limit on the number of archive request per second.                                                                                                                                                                          | 300           |
```

Comment:
plz remove this config. Starting from 1.20 new archival implementation (which is used by default) won't use this config.

Reply:
removed